configure

```diff
@@ -1545,7 +1543,8 @@ Optional Packages:
   --with-perl             build Perl modules (PL/Perl)
   --with-python           build Python modules (PL/Python)
   --with-java             build Java modules (PL/Java)
-  --with-r             build R modules (PL/R)
+  --with-r                build R modules (PL/R)
```

Comment:
--with-perl, ... follows postgresql though I agree with you but we can keep it for tradition.  pgcypto typo, yes, thanks.